client/src/features/projectsV2/fields/NameFormField.tsx

```diff
@@ -51,9 +52,12 @@ export default function NameFormField<T extends FieldValues>({
         rules={{ required: true, maxLength: 99 }}
       />
       <div className="invalid-feedback">Please provide a name</div>
-      <FormText id={`${entityName}NameHelp`} className="input-hint">
-        The name you will use to refer to the {entityName}.
-      </FormText>
+      {helpText && typeof helpText === "string" && (
+        <FormText id={`${entityName}-help`} className="input-hint">
+          {helpText}
+        </FormText>
+      )}
+      {helpText && typeof helpText !== "string" && <>{helpText}</>}
```

Comment:
Can we avoid doing a special case here? Also, the association breaks in the second case (no `aria-describedby`).

```suggestion
      {helpText ? (
        <FormText id={`${entityName}-help`}>
          {helpText}
        </FormText>
      ) : (
        <FormText id={`${entityName}NameHelp`}>
          The name you will use to refer to the {entityName}.
        </FormText>
      )}
```

Note: we should NOT use the `input-hint` class since it comes from Renku 1.0 css files which pollute the global CSS namespace. For reference, see `client/src/styles/renku_additional_classes.scss` where only a couple of CSS classes are added to the global v2 styles.

It is OK to define a `input-hint` class in a CSS/SCSS module to be imported widely, as the corresponding classes would be prefixed.